infra/gcp/ensure-prod-storage.sh

```diff
@@ -404,4 +404,12 @@ color 6 "Handling special cases"
         "${PROD_PROJECT}"
 ) 2>&1 | indent
 
+    # Special case: don't use retention on vulnerability dashboard bucket
+    #               'ci-release-vulndash-update' runs periodically in Prow and
+    #               requires access to overwrite the dashboard's html.
+    #               This should maybe one day be wired up as a Netlify site, but
+    #               one step at a time!
+    color 6 "Removing retention on the ${PROD_PROJECT}-vuln-dashboard bucket"
+    gsutil retention clear "gs://${PROD_PROJECT}-vuln-dashboard"
```

Comment:
I think this should be inside the "handling special cases" block above?

Reply:
Fixed!